### PR TITLE
fix: NuGet symbols not published (#3954)

### PR DESCRIPTION
**Pull request by aloker**
Updates the build script to handle the newer snupkg format.

Please try and check if this works - can't test it with the official nuget packages obviously.

refs #3954

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -486,17 +486,15 @@ Target "PublishNuget" (fun _ ->
     if (shouldPushNugetPackages || shouldPushSymbolsPackages) then
         printfn "Pushing nuget packages"
         if shouldPushNugetPackages then
-            let normalPackages= 
-                !! (outputNuGet @@ "*.nupkg") 
-                -- (outputNuGet @@ "*.symbols.nupkg") |> Seq.sortBy(fun x -> x.ToLower())
+            let normalPackages= !! (outputNuGet @@ "*.nupkg") |> Seq.sortBy(fun x -> x.ToLower())
```
Comment (aloker) [inline build.fsx]: No .symbols.nupkg files are created anymore, just .nupkg and .snupkg.

File: build.fsx [hunk 2]
```diff
@@ -486,17 +486,15 @@ Target "PublishNuget" (fun _ ->
...
             for package in normalPackages do
                 try
-                    publishPackage (getBuildParamOrDefault "nugetpublishurl" "") (getBuildParam "nugetkey") 3 package
+                    publishPackage (getBuildParamOrDefault "nugetpublishurl" "https://api.nuget.org/v3/index.json") (getBuildParam "nugetkey") 3 package
```
Comment (aloker) [inline build.fsx]: This one's crucial: auto pushing related snupkg files only works with the V3 API.

Comment (Aaronontheweb) [reply build.fsx]: Been meaning to make that change myself - we should probably also update the `build-system/windows-release.yaml` file  too.

Comment (aloker) [reply build.fsx]: Good point, PR updated

**Comment by Aaronontheweb** [comment]
@aloker one last change:

```
let shouldPushSymbolsPackages = (hasBuildParam "symbolspublishurl") && (hasBuildParam "symbolskey")
```

Is there ever a condition where we wouldn't want to try to push debug symbols? Is this check necessary any more?

**Comment by aloker** [comment]
If auto-publishing the related snupkg files works, the check and the whole second part (publishing snupkg files) isn't necessary. I'm not sure if auto-publishing always works for private package servers or if this hardcoded for nuget.org only. I don't have a private package server at hand that's uses the V3 API.

Personally I'd leave it in just in case someone publishes to a private package server and needs to push the symbols separately.

**Pull request merged**